thaomonster/rancid-tomatillos
# Tm/feature/movie details

### What does this PR do? 
This PR completes iteration 2, adding the movie detail page and displaying the information.

### Is this a feature or Fix/Refactor?  
Feature

### Problems Encountered & Resulting Solutions  
N/A

### Where should the reviewer start?
src/Data/movieData.js lines: 3-15;
src/components/App/App.js lines: 17-20, 24-34;
src/components/Header/Header.js lines: 3, 6;
src/components/MovieDetail/MovieDetail.js lines 1-27;
src/components/Poster/Poster.js lines 4, 6;
src/components/MovieDetail/MovieDetail.css lines 1-14;

### How should this be tested?
cd into the project, run npm start and play around with the page, make sure to click on the first movie title to see the detail page for the mock data.

### Steps To Take
Refactor some of the code after we get fetch our API and see what is different.
